video/beta/src/main/java/com/example/video/TextDetection.java

```diff
@@ -0,0 +1,168 @@
+          .addFeatures(Feature.TEXT_DETECTION)
+          .build();
+
+      // asynchronously perform object tracking on videos
+      OperationFuture<AnnotateVideoResponse, AnnotateVideoProgress> response =
+          client.annotateVideoAsync(request);
+
+      System.out.println("Waiting for operation to complete...");
+      // The first result is retrieved because a single video was processed.
+      VideoAnnotationResults result = response.get(300, TimeUnit.SECONDS)
+          .getAnnotationResults(0);
+
+      // Get only the first annotation for demo purposes.
+      TextAnnotation textAnnotation = result.getTextAnnotations(0);
+
+      System.out.println("Text: " + textAnnotation.getText());
+
+      // Get the first text segment.
+      TextSegment textSegment = textAnnotation.getSegments(0);
+
+      System.out.println(String.format("Start time: %.2f",
+          textSegment.getSegment().getStartTimeOffset().getSeconds()
+              + textSegment.getSegment().getStartTimeOffset().getNanos() / 1e9));
+
+      System.out.println(String.format("End time: %.2f",
+          textSegment.getSegment().getEndTimeOffset().getSeconds()
+              + textSegment.getSegment().getEndTimeOffset().getNanos() / 1e9));
+
+      System.out.println("Confidence: " + textSegment.getConfidence());
+
+      // Show the first result for the first frame in the segment.
+      TextFrame textFrame = textSegment.getFrames(0);
+
+      System.out.println("Time offset for the first frame: "
+          + (textFrame.getTimeOffset().getSeconds()
+          + textFrame.getTimeOffset().getNanos() / 1e9));
+
+      System.out.println("Rotated Bounding Box Vertices:");
+      for (NormalizedVertex normalizedVertex :
+          textFrame.getRotatedBoundingBox().getVerticesList()) {
+        System.out.println(String.format("\tVertex.x: %.2f, Vertex.y: %.2f",
+            normalizedVertex.getX(),
+            normalizedVertex.getY()));
+      }
+
+      return result;
+    }
+  }
+  // [END video_detect_text_beta]
+
+  // [START video_detect_text_gcs_beta]
+  /**
+   * Detect Text in a video.
+   *
+   * @param gcsUri the path to the video file to analyze.
+   */
+  public static VideoAnnotationResults detectTextGcs(String gcsUri) throws Exception {
+
+    try (VideoIntelligenceServiceClient client = VideoIntelligenceServiceClient.create()) {
+
+      // Create the request
+      AnnotateVideoRequest request = AnnotateVideoRequest.newBuilder()
+          .setInputUri(gcsUri)
+          .addFeatures(Feature.TEXT_DETECTION)
+          .build();
+
+      // asynchronously perform object tracking on videos
+      OperationFuture<AnnotateVideoResponse, AnnotateVideoProgress> response =
+          client.annotateVideoAsync(request);
+
+      System.out.println("Waiting for operation to complete...");
+      // The first result is retrieved because a single video was processed.
+      VideoAnnotationResults result = response.get(300, TimeUnit.SECONDS)
```

Comment:
Separate `response.get` from `.getAnnotationResults` (same as example above)

In general, it's more readable to break up into two statements on two lines instead of 1 statement over 2 lines.

Reply:
Fixed